### ISSUE TITLE
files and dir cleanup

### ISSUE THREAD

**Issue by freisei** [linuxcnc-probe-screen/probe-screen-ng#3]
Before beginnig work and doing code-commits i want to take the chance and doing a big cleanup.

I tried to move anything that is only psng (probe-screen-next-generation) relevant into the psng-subdir.
Anything the user must touch/configure is in the main-config dir.

Anything which can´t be moved out of standard-dirs has prpended "psng_"

```

freisei@linuxcnc:/usr/local/src/probe-screen-ng$ tree |grep -v png
.
└── probe-screen
    ├── LICENSE.md
    ├── macros
    │   ├── psng_block_down.ngc
    │   ├── psng_down.ngc
    │   ├── psng_gotots.ngc
    │   ├── psng_manual_change.ngc
    │   ├── psng_probe_down.ngc
    │   ├── psng_xminus.ngc
    │   ├── psng_xplus.ngc
    │   ├── psng_yminus.ngc
    │   └── psng_yplus.ngc
    ├── psng
    │   ├── icons
    │   ├── icons_1024x768
    │   │   ├── probe_screen.glade
    │   │   ├── probe_screen.hal
    │   ├── probe_screen.glade
    │   ├── probe_screen.py
    │   └── probe_screen_scroll.glade
    ├── psng.hal
    ├── psng.ini
    ├── psng-postgui.hal
    ├── python
    │   ├── remap.py
    │   ├── stdglue.py
    │   └── toplevel.py
    ├── README.md

6 directories, 148 files
```

**Comment by kiall** [comment]
`psng/icons_1024x768/probe_screen.glade` should probably move next to the other glade files as `psng/probe_screen_1024x786.glade` (easier to find, harder to forget to update)

is `psng/icons_1024x768/probe_screen.hal` a duplicate of `psng.hal`? (There is currently 2 copies of that file in the repo).. if it is, we can delete it.

Otherwise, LGTM based on the tree above!

**Comment by freisei** [comment]
branch files_cleanup should be in working state now. 
If anyone could do some testing, it would be very useful. #

**Comment by kiall** [comment]
Overall, looks good - I can't test at the moment as I'm part way through swapping out the control board in my panel.

Can you open a PR for the change? Much easier to review and comment in the PR :)

**Comment by freisei** [comment]
OK, as you see i am new to github :)

**Comment by freisei** [comment]
> `psng/icons_1024x768/probe_screen.glade` should probably move next to the other glade files as `psng/probe_screen_1024x786.glade` (easier to find, harder to forget to update)
> 
> is `psng/icons_1024x768/probe_screen.hal` a duplicate of `psng.hal`? (There is currently 2 copies of that file in the repo).. if it is, we can delete it.

DONE.

**Comment by kiall** [comment]
All done & merged in #7 :)
